bin/cheribsdtest/cheribsdtest.c

```diff
@@ -721,9 +657,6 @@ main(int argc, char *argv[])
 		case 's':
 			sleep_after_test = 1;
 			break;
-		case 'u':
```

Comment:
I wonder if we should add a temporary `errx(1, "-u is no longer supported due to libcheri removal");` incase people still have `-u`'s in CI bits.

Reply:
I mean, we're the only ones with CheriBSD in CI, and I doubt anyone other than us has even discovered -u...